Workbooks/Insights/ADXCluster/Insights.workbook

```diff
@@ -0,0 +1,1466 @@
+                      },
+                      "emptyValCustomText": "-"
+                    }
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Export health and performance-ContinuousExportResult Timeline",
+                    "formatter": 5,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Export health and performance-ContinuousExportResult",
+                    "formatter": 1,
+                    "formatOptions": {},
+                    "numberFormat": {
+                      "unit": 17,
+                      "options": {
+                        "style": "decimal",
+                        "useGrouping": false,
+                        "maximumFractionDigits": 1
+                      },
+                      "emptyValCustomText": "-"
+                    }
+                  }
+                ],
+                "rowLimit": 10000,
+                "filter": true,
+                "hierarchySettings": {
+                  "treeType": 1,
+                  "groupBy": [
+                    "Subscription"
+                  ],
+                  "expandTopLevel": true,
+                  "finalBy": "Name"
+                },
+                "labelSettings": [
+                  {
+                    "columnId": "Subscription"
+                  },
+                  {
+                    "columnId": "Name"
+                  },
+                  {
+                    "columnId": "microsoft.kusto/clusters-Export health and performance-ExportUtilization",
+                    "label": "Export Utilization (Max)"
+                  },
+                  {
+                    "columnId": "microsoft.kusto/clusters-Export health and performance-ExportUtilization Timeline",
+                    "label": "Export Utilization Timeline"
+                  }
+                ]
+              },
+              "sortBy": [],
+              "showExportToExcel": true
+            },
+            "conditionalVisibility": {
+              "parameterName": "selectedTab",
+              "comparison": "isEqualTo",
+              "value": "Export"
+            },
+            "showPin": true,
+            "name": "Export Performance"
+          },
+          {
+            "type": 10,
+            "content": {
+              "chartId": "workbook19356bdc-cfd5-47c7-8e70-05bbd3f39558",
+              "version": "MetricsItem/2.0",
+              "size": 2,
+              "chartType": 0,
+              "metricScope": 0,
+              "resourceIds": [
+                "{Resources}"
+              ],
+              "timeContext": {
+                "durationMs": 0
+              },
+              "timeContextFromParameter": "TimeRange",
+              "resourceType": "microsoft.kusto/clusters",
+              "resourceParameter": "Resources",
+              "metrics": [
+                {
```

Comment:
if we move this metric object to the end of the metrics array, we can display it at the end of the columns